spec/support/seeds/seeds_e2e.rb

```diff
@@ -0,0 +1,87 @@
+return if Rails.env.production?
```

Comment:
I have both because I'm paranoid of this running on prod. If you think it's fine to have just `return unless Rails.env.test? && ENV["E2E"].present?`, let me know.

Reply:
I think about it like this: If you were to put the env check for `test` first, the second guard clause can never be `true`. I think that is a strong indication  that the check is unnecessary.

Reply:
I figured as well. Just wanted someone to invalidate my paranoia. 😎